nservicebus/messaging/routing-extensibility.md

```diff
@@ -34,4 +34,4 @@ snippet: endpoint-mapping-configurationprovider
 
 The `MessageEndpointMappings` collection can be populated based on any external source. It is read during the endpoint start-up, before any messages are sent.
 
-NOTE: The route table is not updated during run-time, even if the contents of the mappings collection change. In case the routing data changes frequently, consider implementing a mechanism that would restart the endpoint when the change is detected.
\ No newline at end of file
+NOTE: The route table is not updated during run-time, even if the contents of the mappings collection change. In case the routing data changes frequently, consider implementing a mechanism that would restart the endpoint when the change is detected.
```

Comment:
I was using the Edit button and github did the rest. No clue, why it thinks here is a change